### ISSUE TITLE
[BUG] Access to database is denied could not find driver

### ISSUE THREAD

**Issue by m4dz** [FreshRSS/FreshRSS#3528]
**Describe the bug**
Starting with a Fresh install, call the `./cli/do-install.php` is leading to the following error:

```
Access to database is denied for ``: could not find driver
```

**To Reproduce**
1. Get a new fresh install of `1.18.0`:
   ```
   $ mkdir freshrss
   $ cd freshrss
   $ wget -O- https://github.com/FreshRSS/FreshRSS/archive/1.18.0.tar.gz | tar -xz --strip-components=1
   ```
2. Prepare the environment
   ```
   $ ./cli/prepare.php
   ```
3. Call `do-install` with parameters:
   ```
   $ ./cli/do-install.php --default_user admin --auth_type form --environment production --base_url https://$INSTALL_URL --language $LANGUAGE --title "$TITLE" --db-type mysql --db-host ${DATABASE_HOST}:3306 --db-user ${DATABASE_USERNAME} --db-password ${DATABASE_PASSWORD} --db-base ${DATABASE_NAME}
   ``` 
4. Get the following output
   ```
   FreshRSS install…
   FreshRSS database error: Access to database is denied for ``: could not find driver
   ```

**Expected behavior**
Normal install

**Environment information (please complete the following information):**
 - OS: [e.g. Debian Buster
 - FreshRSS version: 1.18.0
 - Database version: MariaDB 10.5
 - PHP version: 7.4
 - Installation type: manual

**Additional context**
The problem occurs since release 1.18.0. Same commands with release 1.17.0 report no issues.

**Comment by Alkarex** [comment]
Could you please provide the output of:

```sh
php -i | grep -i mysql

php -i | grep -i PDO
```

**Comment by online-ua** [comment]
You need to enable pdo_sqlite...

**Comment by m4dz** [comment]
> You need to enable pdo_sqlite...

That's it! Thanks.

Why do we need `pdo_sqlite` enabled even when using explicitely mysql as DB?

**Comment by Alkarex** [comment]
@m4dz Should be fixed by https://github.com/FreshRSS/FreshRSS/pull/3544
Tests welcome

**Comment by m4dz** [comment]
> @m4dz Should be fixed by #3544
> Tests welcome

Thanks for the fix, works perfectly fine 👍
